aux/bootstrap/flake.nix

```diff
@@ -0,0 +1,20 @@
+{
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable-small";
+
+  inputs.disko = {
+    url = "github:nix-community/disko";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  inputs.nixos-generators = {
+    url = "github:nix-community/nixos-generators";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+
+  inputs.nixos-anywhere = {
+    url = "github:numtide/nixos-anywhere?ref=refs/pull/213/head";
```

Comment:
Should be only merged after upstream is merged and updated to avoid short-term incremental maintenance.

Reply:
Next PR incoming. Benefit of waiting no longer outweighs the cost of holding back on the subsequent PR.